src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18193.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue18193 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] Navigation doesn't work on sixth tab in shell";
+
+		public Issue18193(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void ShellNavigationShouldWorkInMoreTab()
+		{
+			App.WaitForElement("NavigationToPage6Button");
+			App.Click("NavigationToPage6Button");
```

Comment:
Here, could navigate back and use VerifyScreenshot to validate the "More" menu. 
![image](https://github.com/user-attachments/assets/44395d42-1b14-4e26-becf-27807a3c5021)
Then, navigate again to the Page 6 and continue testing, navigating to the detail page and Page 2.